bundles/org.openhab.binding.emotiva/src/main/java/org/openhab/binding/emotiva/internal/dto/EmotivaAckDTO.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.emotiva.internal.dto;
+
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAnyElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * The EmotivaAck message type. Received from Emotiva device whenever a {@link EmotivaControlDTO} with
+ * {@link EmotivaCommandDTO} is sent.
+ *
+ * @author Espen Fossen - Initial contribution
+ */
+@XmlRootElement(name = "emotivaAck")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class EmotivaAckDTO {
+
+    @XmlAnyElement(lax = true)
+    private List<Object> commands;
+
+    @SuppressWarnings("unused")
+    public EmotivaAckDTO() {
+    }
```

Comment:
Personal preference i guess, bit this constructor is not used, so why add it and later on supress the warning? 
Just removing would be best. Applies to all other DTO's

Reply:
If there is no empty constructor for each DTO, the `JAXBContext.newInstance(EmotivaAckDTO.class, ...` in EmotivaXmlUtils:75 will fail with the following

```
com.sun.xml.bind.v2.runtime.IllegalAnnotationsException: 1 counts of IllegalAnnotationExceptions

org.openhab.binding.emotiva.internal.dto.EmotivaAckDTO does not have a no-arg default constructor.
	this problem is related to the following location:
		at org.openhab.binding.emotiva.internal.dto.EmotivaAckDTO
			at 
			
com.sun.xml.bind.v2.runtime.IllegalAnnotationsException$Builder.check(IllegalAnnotationsException.java:76)
	at com.sun.xml.bind.v2.runtime.JAXBContextImpl.getTypeInfoSet(JAXBContextImpl.java:450)
	at com.sun.xml.bind.v2.runtime.JAXBContextImpl.<init>(JAXBContextImpl.java:282)
	at com.sun.xml.bind.v2.runtime.JAXBContextImpl.<init>(JAXBContextImpl.java:109)
	at com.sun.xml.bind.v2.runtime.JAXBContextImpl$JAXBContextBuilder.build(JAXBContextImpl.java:1142)
	at com.sun.xml.bind.v2.ContextFactory.createContext(ContextFactory.java:144)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:251)
	at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:240)
	at javax.xml.bind.ContextFinder.find(ContextFinder.java:363)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:691)
	at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:632)
	at org.openhab.binding.emotiva.internal.protocol.EmotivaXmlUtils.<init>(EmotivaXmlUtils.java:75)
	at org.openhab.binding.emotiva.internal.AbstractDTOTestBase.<init>(AbstractDTOTestBase.java:28)
	at org.openhab.binding.emotiva.internal.protocol.EmotivaXmlUtilsTest.<init>(EmotivaXmlUtilsTest.java:35)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at java.base/java.util.Optional.orElseGet(Optional.java:364)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

```

Reply:
Ah, this somewhat hidden. Thanks in that case leave the constructors as is.